styles/tables/mod.css

```diff
@@ -1,44 +1,59 @@
+/* Modernized table styles */
+
 table {
-  display: block;
+  width: 100%;
   max-width: 100%;
   margin: 0 auto 1rem;
   border-collapse: collapse;
-  border-spacing: 0;
-  inline-size: fit-content;
   overflow-x: auto;
+  border-radius: 0.5rem;
+  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.1);
 }
 
 .table-responsive {
-  display: grid;
-  grid-template-columns: repeat(auto-fit, minmax(0, 1fr));
+  display: block;
+  width: 100%;
```

Comment:
The `display: grid;  grid-template-columns;` cannot be removed actually, because it'll "break" the body size on small screens on many browsers as seen below:
![image](https://github.com/lowlighter/matcha/assets/22963968/298e025d-94ed-47b5-8e28-eab9fee349d2)

For some reason even with `overflow: auto` this issue occurs, unless a better style exists to fix this issue, I think this should be kept (which include the removed block below)